splinepy/bspline.py

```diff
@@ -179,6 +179,63 @@ def determine_new_knots(kv_unique, n_knots):
             )
             self.insert_knots(para_dim, new_knots)
 
+    def uniform_refine_fixed_knots(self, para_dims=None, total_knots=4):
```

Comment:
This function isn't clear and I don't think it should be part of API. Please remove, or tell me why this is crucial

Reply:
Removed